app/backend/src/main/kotlin/com/gamelounge/backend/service/AccessService.kt

```diff
@@ -101,4 +106,38 @@ class AccessService (
         return true
     }
 
+    fun forgotPassword(username: String,email: String) {
+        //validate username and email
+        val user = userRepository.findByUsername(username)
+            ?: throw UsernameNotFoundException("User not found.")
+        if (user.email != email){
+            throw EmailNotFoundException("Email not found.")
+        }
+        //generate token
+        val token = passwordResetTokenService.createToken(user)
+        //send email
+        emailService.sendSimpleMessage(email, "Password Reset", "Click the link to reset your password: http://localhost:8080/reset-password?token=$token")
```

Comment:
The link to send to the e-mail cannot be a localhost link. This needs to be parameterized. Maybe you can solve it by adding a key value to application.properties.

Reply:
ok it is fixed now